### PR TITLE
Add shared_roster_SUITE to dynamic_domains

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/73606) / small_tests / f2ef20986ed1a78f79654b704162757bb08480c8
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3282/73606/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3282/73606/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-16_18.53.05/lib.mongooseim.logs/run.2021-09-16_18.53.06/suite.log.html)

File: big_tests/tests/shared_roster_SUITE.erl [hunk 1]
```diff
@@ -143,7 +142,7 @@ add_user(Config) ->
 
 start_roster_module(ldap) ->
     case rpc(mim(), gen_mod, start_module,
-             [ct:get_config({hosts, mim, domain}), mod_shared_roster_ldap, get_ldap_args()]) of
+             [ct:get_config({hosts, mim, host_type}), mod_shared_roster_ldap, get_ldap_args()]) of
```
Comment (vkatsuba) [inline big_tests/tests/shared_roster_SUITE.erl:145]: Should be used `-import(domain_helper, [host_type/0]).`

Comment (chrzaszcz) [reply big_tests/tests/shared_roster_SUITE.erl:145]: Yes, please do

**Review by chrzaszcz — COMMENTED** [review]
The main issue with this PR is that `mod_shared_roster_ldap` does not support dynamic domains. CI is green because it requires LDAP and there is no LDAP in the `dynamic_domains` preset (Postgres is used), so the whole suite is just skipped, see https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3282/73611/pgsql_mnesia.24.0-1/big/ct_run.test@default-bb0188bb-c20e-49e5-8b29-b620dcec34d5.2021-09-16_19.13.07/big_tests.tests.shared_roster_SUITE.logs/run.2021-09-16_19.23.38/suite.log.html

**Comment by vkatsuba** [comment]
> The main issue with this PR is that `mod_shared_roster_ldap` does not support dynamic domains. CI is green because it requires LDAP and there is no LDAP in the `dynamic_domains` preset (Postgres is used), so the whole suite is just skipped, see https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3282/73611/pgsql_mnesia.24.0-1/big/ct_run.test@default-bb0188bb-c20e-49e5-8b29-b620dcec34d5.2021-09-16_19.13.07/big_tests.tests.shared_roster_SUITE.logs/run.2021-09-16_19.23.38/suite.log.html

Thanks. Yes can be. Basically by some reason I cannot check it locally and I was try to push it for check it in CI side.

**Comment by vkatsuba** [comment]
Closed due to that `mod_shared_roster_ldap` does not support dynamic domains.

**Pull request closed without merging**